src/compile/layout.ts

```diff
@@ -61,7 +61,7 @@ function getCellHeight(model: Model): LayoutValue {
       return model.config().cell.height;
     }
   }
-  return model.fieldDef(Y).scale.bandWidth;
+  return 21 /* config.scale.bandWidth */;
```

Comment:
It was original set in combination with positionFieldDef's definition.   (Have to be the same for both x and y) 
  
But now x scale has flexible bandWidth and going forward, we should move these default values to `config.scale.*` anyway so I decide not to bother with `encoding.x|y.scale` `s default.